src/pages/about.js

```diff
@@ -0,0 +1,18 @@
+import React from "react"
```

Comment:
Do we need this page? I think it's not necessary, and think we could remove it, unless I'm misremembering something.